.github/workflows/tests-gpu.yml

```diff
@@ -5,7 +5,12 @@ name: Tests on GPU
 # or similar.
 
 on:
-    [push, pull_request]
+  pull_request:
```

Comment:
```suggestion
  # Limit this workflow to pushes and PRs to particular branches (e.g. main or
  # release branches) so that the workflows don't run and automatically fail on
  # branches in forked repositories (where there is typically no `gpu` worker to
  # run on).
  pull_request:
```